tests/integration/targets/rds_instance/roles/rds_instance/tasks/test_states.yml

```diff
@@ -237,26 +256,19 @@
         id: "{{ instance_id }}"
         state: absent
         skip_final_snapshot: True
+        wait: False
       register: result
 
     - assert:
         that:
           - result.changed
 
-    - name: Ensure instance was deleted
-      rds_instance_info:
-          db_instance_identifier: '{{ instance_id }}'
-      register: db_info
-
-    - assert:
-        that:
-          - db_info.instances | length == 0
```

Comment:
I guess we should keep this. Since  rds_instance_info: is aliased, we should also include some tests using rds_instance_info.

Reply:
Good point. I removed to speed up this tests since its the longest one, but youre right we do need it